### PR TITLE
! Fix font colors and padding

### PR DESCRIPTION
**Pull request by wfortin**
* Use text-color for darker text
* Clean up padding conflict with table style rules
image

### CODE REVIEW

**Comment by JPLachance** [comment]
Look's good!

File: scss/lib/datepicker.scss [hunk 1]
```diff
@@ -219,9 +219,9 @@ tbody.datepickerDays td.datepickerSelected {
   background-color: #0077cc;
 }
 
-/* Grey out dates in the future */
-div.datepicker td.datepickerFuture a {
-  color: #dddddd;
+/* yellow cell background on hover */
+div.datepicker tbody.datepickerDays td:hover {
+  background-color: #ffcc33;
```
Comment (pochretien) [inline scss/lib/datepicker.scss]: can extract this color in variable or its only used for this datepicker ?

**Comment by pochretien** [comment]
:shipit:

File: scss/lib/datepicker.scss (hunk 1 above)
Comment (wfortin) [inline scss/lib/datepicker.scss]: It's the base libraries colors, we could clean up the unused classes someday.

**Pull request merged**